packages/app/src/specs/InlineSpecList.cy.tsx

```diff
@@ -222,5 +222,30 @@ describe('InlineSpecList', () => {
       mountInlineSpecList({ experimentalRunAllSpecs: true, specFilter: 'B' })
       hoverRunAllSpecs('src/components', 1)
     })
+
+    it('allows keyboard interactions to run spec groups without toggling sections', () => {
+      mountInlineSpecList({ experimentalRunAllSpecs: true })
+
+      const mutationStub = cy.stub().as('mutationStub')
+
+      cy.stubMutationResolver(RunAllSpecsDocument, () => {
+        mutationStub()
+      })
+
+      cy.findAllByTestId('spec-file-item').should('have.length', 4)
+      cy.findAllByTestId('run-all-specs-button').eq(0)
+      .click()
+      .type(' ')
+
+      // make sure typing didn't change displayed items
+      cy.findAllByTestId('spec-file-item').should('have.length', 4)
+      cy.findAllByTestId('run-all-specs-button').eq(1)
+      .focus()
+      .type('{enter}')
+
+      // make sure typing didn't change displayed items
+      cy.findAllByTestId('spec-file-item').should('have.length', 4)
+      cy.get('@mutationStub').should('have.been.calledThrice')
```

Comment:
I must be missing something - I see the `space` and `enter` invocations of Run All, what's the third?

Reply:
Before the space, there's a click which does the same thing. Then it already has focus for the type. Then for `{enter}` we artificially focus, to avoid Cypress doing it's own click.

Not sure I really need to test the click here as such, but it's sorta nice to assert all 3 in one test.